app/helpers/format-currency.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+const { Helper } = Ember;
+
+export function formatCurrency(params) {
+  let value = 0,
+      sign = '';
+  sign = params[0];
```

Comment:
Let the sign be the second parameter. Since it's optional. Keep the default sign as `$`